mtconnect-server/src/main/java/io/github/protocol/mtconnect/server/MtHandler.java

```diff
@@ -0,0 +1,6 @@
+package io.github.protocol.mtconnect.server;
+
+import io.github.shoothzj.http.facade.server.RequestHandler;
+
+public interface MtHandler extends RequestHandler {
```

Comment:
I am not sure this's a good idea. Normally, we would make MtHandler has multi protocol specified method.

If I want to add a http path, what sould I do?

Reply:
make `MtRequestProcessor` to extends `RequestHandler`